packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/pools.component.js

```diff
@@ -0,0 +1,18 @@
+import template from './pools.html';
+import controller from './pools.controller';
+
+export default {
+  bindings: {
+    projectId: '<',
+    databaseId: '<',
+    database: '<',
+    postgresDatabases: '<',
+    trackDashboard: '<',
+    pools: '<',
+    goBackToPools: '<',
+    databasesTrackPrefix: '<',
```

Comment:
You should not inject `databasesTrackPrefix` here, as `trackDashboard` function is made to handle tracking